### PR TITLE
Genotype clones fixes

### PR DESCRIPTION
**Pull request by andremrsantos**
Hi,
upon reviewing the `genotypeClones.py` code results I noticed two bugs that I commit solutions here:

- `computeKeyRate` was using the BCs total weight instead of the edge weight to estimate the transfections rate. I included a `source`  parameter to determine the edges and correctly compute the rates.
- `remove_edges` was excluding edges twice or not decreasing the node's total weight of some edges were being excluded making the total weight inconsistent with the sum of the weight of the edges. As `G.edges` emit edges on arbitrary order (either (BC, cellBC) or (cellBC, BC)) sometimes they were not recognized as included to `edgesToRemove`. To fix this I've added the reverted order to `edgesToRemove` which accounts for either cases and guarantees consistency.

### CODE REVIEW

File: transposon/genotypeClones.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ def pruneOrphanNodes(G):
 def remove_edges(G, edgesToRemove):
     #Convert to set to speed-up inclusion check
     edgesToRemove = set(edgesToRemove)
+    edgesToRemove.update([(y, x) for (x, y) in edgesToRemove])
```
Comment (mattmaurano) [inline transposon/genotypeClones.py:147]: Andre, can you add a comment to expand on what this is doing? This might be better done in a separate PR so it doesn't show up with the unrelated commit below.

Comment (andremrsantos) [reply transposon/genotypeClones.py:148]: I've committed them separately, but do you want me to convert this to two distinct PR?

Comment (mattmaurano) [reply transposon/genotypeClones.py:148]: yes, in the future it's best to make separate PRs. That way I can squash and merge regardless of how many commits happen.
